src/features/MovieListTile/styled.js

```diff
@@ -1 +1,184 @@
-import styled from "styled-components";
\ No newline at end of file
+import styled from "styled-components";
+import { ReactComponent as StarIcon } from "../../icons/star.svg";
+
+export const StyledMovieDetailsTileList = styled.div`
+  max-width: 1368px;
+  box-shadow: 4px 4px 10px #bac7d580;
+  background-color: ${({ theme }) => theme.color.white};
+  display: grid;
+  grid-gap: 0px;
+  align-content: start;
+  grid-template-columns: 324px;
+  grid-template-rows: auto auto;
+  margin-bottom: 50px;
+  row-gap: 16px;
+
+  @media (max-width: 1080px) {
```

Comment:
You should try using breakpoints from theme.js to implement max-width of mediaQueries